core/templates/services/user-backend-api.service.ts

```diff
@@ -150,6 +165,67 @@ export class UserBackendApiService {
       data: data
     }).toPromise();
   }
+
+  async loadProfileImage(username: string = ''): Promise<ImageFile> {
+    // Artificially load an image from a random exploration where you
+    // uploaded an image. We'll pretend that it is the profile image
+    // that is retrieved from the backend.
+
+    // Test image 1.
+    const image_1 = this.assetsBackendApiService.loadImage(
+      AppConstants.ENTITY_TYPE.EXPLORATION, 'lOZraTUa1x8b',
+      'img_20221108_210419_1pse2cowr9_height_275_width_237.png');
+
+    // Test image 2.
+    const image_2 = this.assetsBackendApiService.loadImage(
+      AppConstants.ENTITY_TYPE.EXPLORATION, 'sIW3TNaiFWQD',
+      'img_20221108_111758_k26g4dj4h7_height_201_width_203.png');
+
+    username = username || await this.getUserInfoAsync().then(
+      userInfo => userInfo.getUsername());
+    // All of this logic is just for testing.
+    if (username === 'eric') {
+      return image_1;
+    } else {
+      return image_2;
+    }
+    // ^^^ Delete everything above this line once merged with Hitesh's changes.
```

Comment:
Everything above this line will be deleted and was just used for my own testing.